### ISSUE TITLE
create a one-liner script for NetData installation

### ISSUE THREAD

**Issue by Erkan-Yilmaz** [Erkan-Yilmaz/Gridcoin-tasks#112]
< @Peppernrino > possibly create a new task: create a one-liner script for netdata installation

see: https://github.com/grctest/GRC-Netdata

**Comment by Peppernrino** [comment]
it would be easier to set up if it were like the AutoNode script.

also, good to put here that not everybody used grc as an alias during setup of the node. @grcjamezz had trouble getting it to work on his nodes with an alias. i'll post in IRC for him to clarify, but it caused problems for me as well.

**Comment by grctest** [comment]
Netdata is already a 2-line setup, it's really simple.
Installation of GRC-Netdata is a bit more hands-on, sure, but it's still heavily a work-in-progress.

**Comment by Peppernrino** [comment]
1 is less than 2. checkmate.

**Comment by Peppernrino** [comment]
i wasn't trying to hack on your efforts... @Erkan-Yilmaz just puts so much pressure on us to create these new tasks. ;)

**Comment by grctest** [comment]
@Foggyx420 has significantly improved the GRC-Netdata repo, including some automated installation lines. I think we're getting close to this task being completed.

**Comment by Foggyx420** [comment]
yes there is a more automated script at the moment thou markets are still seperate. I'm working to make a installer script that can accept user inputs and check enviroment to make sure it is setup correctly to accept the install as well as install the required packages. also plan to allow other distros in near future to be able to install as well. make it more platform friendly. once ubuntus base installer that im currently working on is done that will be 100% automated plus some user inputs as well. and ofcourse checks to make sure it will install correctly. more user friendly is my term

**Comment by Foggyx420** [comment]
My script has successfully installed on Ubuntu and Debian should be much the same but still need to be tested on other releases.  The goal is to have this replace the current GRC-Netdata on @grctest repo. Any future changes from there on out can be pull requested to his repo. This setup includes the basic gridcoin chart + geoip stuff (note on initial start freegeoip can take a short little bit to get updated and sometimes the chart on geo ip information will say a "large number" Other. This is normal as netdata starts faster and the geo ip script has to run on its timer. I may add sleep to it later on but getting this working is important. Also added version so in future it can be modified and least a user can know if they have a current version or not. Still work needs to be done but testing is important for now and enhancements can come later. This was a wee bit difficult as my daughter did delete 800+ lines of code lol. *SIGH*

Testers!
Fedora
Arch
openSUSE
Debian

You must remove a previous GRC-Netdata installation before hand if you test it on a machine that has it already installed. It has different cleaner filenames.

This setup is smaller in the sense of less files and is completely customizable. The systemd setup is also customizable but within limits to not cause problems on a users system via resources.

The other releases I've put into the setup.sh needs to be tested as I've kind of blindly done so based on searches of the packages they would be in as well as the package installer that is common on the release.

The setup.sh is large but It works and lets users inputs as well as verifies various information.

For debugging in the sense of testing one section at a time you can goto the bottom of the bash file and comment out sections your not testing. :)

Message me on here or IRC or head over to https://github.com/Foggyx420 and enter the Netdata-Test repo.

**Comment by Foggyx420** [comment]
Fedora should work now. I tested it on one version of fedora. but could be verified on others.
need Arch and openSUSE testing.
